### PR TITLE
Display calculator

### PR DESCRIPTION
**Pull request by Laylo309**
- [x] Delete all the boilerplate from CRA (text, images, styles).
- [x] Create a directory called components.
- [x] Inside components create a new Calculator.js file.
- [x] In Calculator.js, create a React component that matches this design:
- [x] Don't add any extra design.
- [x] Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- [x] Use class-based components.
- [x] Once the Calculator is ready, import it and use it in your main component (App.js).
- [x] Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,10 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    newCalculator.render()
   );
 }
```
Comment (AshakaE) [inline src/App.js]: The `App.js` component should be class-based.

- [x] Please update this component to use a class-based component

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @Laylo309  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] 
   - [x] Kindly update your `README` file because it is still using the default template that     comes with the creation of a React app 😄 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Laylo309) [reply src/App.js]: Done ✅

**Review by george-shammar — APPROVED** [review]
Hi @Laylo309 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

:heavy_check_mark: Thank you for making the required change. :+1: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**